run-docker.sh

```diff
@@ -5,7 +5,7 @@ set -u
 PS4=" $ "
 set -x
 
-image="$(nix-build -A dockerImage)"
+image="$(nix-build -A eval.config.outputs.containerImage)"
```

Comment:
Again here, what does 'eval' mean?

Reply:
The `evalSpec` function returns an attrset with the form `{ eval = ...; }`.

 - https://github.com/fly-apps/nix-base/blob/e1888f046dda4029841b884694be4e757b125094/support/eval-spec.nix#L29

This allows the `evalSpec function to be expanded in the future if needed. "eval" here would mean "the **evaluated** modules system configuration"